### PR TITLE
[done] Add controller method of task

### PR DESCRIPTION
**Pull request by peoplewhite**

### CODE REVIEW

**Comment by peoplewhite** [comment]
@FunnyQ @s59762 @Dylan0203 
Please verify these and merge. Thanks.

**Comment by FunnyQ** [comment]
methods that in controllers called `actions`

File: app/controllers/tasks_controller.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+  def create
+    @task = Task.create(task_params)
+
+    if @task.save
+      redirect_to :tasks
+    else
+      render :new
+    end
+
```
Comment (FunnyQ) [inline app/controllers/tasks_controller.rb]: 像這種做了某些事情之後 redirect 的 action 最好透過 [flash](http://api.rubyonrails.org/classes/ActionDispatch/Flash.html) 能讓 user 知道執行結果

File: app/views/tasks/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<header>
+<h1>編輯代辦事項</h1>
+</header>
+
+<div class="edit-restaurant-wrapper">
```
Comment (FunnyQ) [inline app/views/tasks/edit.html.erb]: 專業一點啊...class 也改一下吧

File: app/views/tasks/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,31 @@
+<h1>odd daily report</h1>
+
+<div class="create-task-button-wrapper">
+  <%= link_to '建立新代辦事項', new_task_path, class: 'create-button' %>
```
Comment (FunnyQ) [inline app/views/tasks/index.html.erb]: 不是要用 [bootstrap](https://v4-alpha.getbootstrap.com/components/buttons/) 就好嗎？

File: config/routes.rb [hunk 4]
```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
   devise_for :users
   root 'prototype#index'
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  resources :tasks
+  get 'diary' => 'diaries#index'
```
Comment (FunnyQ) [inline config/routes.rb]: why not `resources :diaries`？
就算自己寫，index 應該是 `/diaries` 吧

File: app/controllers/tasks_controller.rb (hunk 1 above)
Comment (FunnyQ) [inline app/controllers/tasks_controller.rb]: http://guides.rubyonrails.org/action_controller_overview.html#the-flash

File: app/views/tasks/edit.html.erb (hunk 2 above)
Comment (peoplewhite) [inline app/views/tasks/edit.html.erb]: @FunnyQ 
solved. Please verify this. Thank you.

**Pull request merged**